types/fibjs/declare/Cipher.d.ts

```diff
@@ -23,121 +23,121 @@
     */
 
 declare class Class_Cipher extends Class__object {
-    
+
     /**
-     * class prop 
+     * class prop
+     *
      *
-     * 
      * @brief 返回当前算法名称
-     * 
+     *
      * @readonly
-     * @type String
+     *
      */
-    
+
     name: string
-    
+
     /**
-     * class prop 
+     * class prop
+     *
      *
-     * 
      * @brief 返回当前算法密码长度，以位为单位
-     * 
+     *
      * @readonly
-     * @type Integer
+     * @note Assumed to be an integer.
      */
-    
+
     keySize: number
-    
+
     /**
-     * class prop 
+     * class prop
+     *
      *
-     * 
      * @brief 返回当前算法初始向量长度，以字节为单位
-     * 
+     *
      * @readonly
-     * @type Integer
```

Comment:
Here, the `@type` actually did add useful information (TypeScript doesn't have a way to represent `number`s that are only ever integers). So I manually preserved the info as a `@note`.